lib/arclight/traject/ead2_config.rb

```diff
@@ -230,10 +230,11 @@
                      parent_id = context.clipboard[:parent].output_hash['id'].first
                      logger.warn('MISSING ID WARNING') do
                        [
-                         "A component in #{parent_id} did not have and ID so one was minted using the #{strategy} strategy.",
+                         "A component in #{parent_id} did not have an ID so one was minted using the #{strategy} strategy.",
                          "The ID of this document will be #{parent_id}#{hexdigest}."
                        ].join(' ')
                      end
+                     hexdigest
```

Comment:
I'm not aware of the discussions which have taken place regarding the minting of IDs using the digest, hence I am hesitant to merge this myself.

Reply:
👍 Gotcha.. good catch @seanaery  looks like we just needed to return it